privacyidea/lib/utils/__init__.py

```diff
@@ -545,52 +546,66 @@ def parse_date(date_string):
 
 def parse_proxy(proxy_settings):
     """
-    This parses the string of the system settings
-    OverrideAuthorizationClient.
+    This parses the string of the system settings OverrideAuthorizationClient into a set of "proxy paths",
```

Comment:
Can you please add some log output ``log.debug`` in this code, so that we can know afterwards, why a certain client IP was identitfied.

Reply:
good idea! I pushed some debug printouts